package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^7.2.3",
+    "cypress": "^3.6.1",
+    "firebase": "^7.3.0",
+    "firebase-tools": "^7.7.0",
```

Comment:
Hadn't heard of `firebase-tools` before. Looks like it's a set of command line tools for interacting with the Firestore database? If so, is it needed/used within the project?

The docs imply that you install it as a global package on your system so you can manage your database. That makes me think you wouldn't add it to your `package.json`, but I could be wrong. Keen to learn what you've learned about it!

Reply:
I added this when I was trying to use cypress-firebase to test the app. It was a step in a tutorial I was following. I will remove!